docs/toolbox/geoanalysis/aggregate_points.md

```diff
@@ -7,11 +7,11 @@ import TabItem from '@theme/TabItem';
 
 # Aggregate Points
 
-The **Aggregate Points** tool performs statistical analysis of points, e.g. **count, sum, min or max**, and aggregates the information on polygons.
+The **Aggregate Points** tool performs statistical analysis of points, e.g. **count, sum, min, or max**, and aggregates the information on polygons.
```

Comment:
In `aggregate_points.md` and `aggregate_polygons.md`, we should elaborate more (or rephrase) this: "... and aggregates the information on polygons." because this is a bit challenging to understand. I assume it means the operation is performed at a polygonal level, or in groups of points as they fall within a polygon, but this is not explained clearly.